docs/tutorial/part-two/index.md

```diff
@@ -248,14 +245,13 @@ import Container from "../components/container"
 console.log(styles)
 ```
 
-The `console.log(styles)` code will log the resulting import so you can see the result of your processed `./about-css-modules.module.css` file. If you open the developer console (using e.g. Firefox or Chrome's developer tools) in your browser, you'll see:
+`console.log(styles)` कोड परिणामी इम्पोर्ट को लॉग करेगा ताकि आप अपने प्रोसेस्ड `about-css-modules.module.css` का परिणाम देख सके। यदि आप अपने ब्राउज़र में डेवलपर कंसोल (उदाहरण के लिए Firefox या Chrome के डेवलपर टूल का उपयोग करके) खोलते हैं, तो आप देखेंगे:
 
-![Import result of CSS module in console](css-modules-console.png)
+![कंसोल में CSS मॉड्यूल का इम्पोर्ट रिज़ल्ट](css-modules-console.png)
 
-If you compare that to your CSS file, you'll see that each class is now a key in the imported object pointing to a long string e.g. `avatar` points to `src-pages----about-css-modules-module---avatar---2lRF7`. These are the class names CSS Modules generates. They're guaranteed to be unique across your site. And because you have to import them to use the classes, there's never any question about where some CSS is being used.
+यदि आप अपनी सीएसएस फ़ाइल से तुलना करते हैं, तो आप देखेंगे कि प्रत्येक क्लास अब एक लंबी स्ट्रिंग के लिए इंगित इंपोर्टेड ऑब्जेक्ट में एक कुंजी है। उदाहरण के लिए `avatar` `src-pages----about-css-modules-module---avatar---2lRF7` को इंगित करता है। ये क्लास नेम सीएसएस मॉड्यूल जनरेट्स करते हैं। वे आपकी साइट पर यूनीक होने की गारंटी देते हैं। और क्योंकि आपको क्लासस का उपयोग करने के लिए उन्हें इम्पोर्ट करना पड़ता है, इसलिए यहाँ कभी कोई सवाल नहीं है कि कुछ सीएसएस का उपयोग कहां किया जा रहा है।
```

Comment:
`ये क्लास नेम सीएसएस मॉड्यूल जनरेट्स करते हैं।` => `सीएसएस मॉडल्स ये क्लास के नाम गेनेराते करता है`

Reply:
you mean ```सीएसएस मॉडल्स ये क्लास के नाम जनरेट्स करता है```

Reply:
Yes. correct.
Actually it should be `जनरेट`

Reply:
yes